src/node/main.ts

```diff
@@ -22,8 +22,8 @@ export const runVsCodeCli = async (args: DefaultedArgs): Promise<void> => {
 
   try {
     await cliProcessMain(args)
-  } catch (error) {
-    logger.error("Got error from VS Code", field("error", error))
+  } catch (error: any) {
```

Comment:
just curious, are there root error types in Node or TypeScript that we can use instead of `any`? seems unfortunate not to have any type information

Reply:
I think it's traditionally been `any` unfortunately. But in TypeScript 4.0, they allow you [to type it as `unknown`](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0/#unknown-on-catch) which they say is safer. But yeah, it's too bad they don't tell us _which_ error this is.